frontend/src/components/auth/Credentials.vue

```diff
@@ -53,6 +53,10 @@ export default {
                 valid = false
                 this.errors.password = 'Required field'
             }
+            if (this.input.password.length > 1024) {
+                valid = false
+                this.errors.password = 'Too long'
+            }
             if (valid) {
```

Comment:
We can remove all the `valid` stuff. Don't think we need it

```suggestion
            if (this.errors.username.length === 0 && this.errors.password.length === 0) {
```

Reply:
I'm fine with a response like: "ZJ, stop over engineering". But felt like a better way than to do the `valid = false` stuff